velox/connectors/hive/HiveDataSource.h

```diff
@@ -54,7 +54,7 @@ class HiveDataSource : public DataSource {
       const std::shared_ptr<common::Filter>& filter) override;
 
   uint64_t getCompletedBytes() override {
-    return ioStats_->rawBytesRead();
+    return ioStats_->read().sum();
```

Comment:
@JkSelf This is zero if we read from cache, is this what we want?

Reply:
@Yuhta Thanks for your review. Yes, based on @oerling  response, it seems that we need to update `rawBytesRead` in the `loadData` function as well. Currently, there appear to be two ways of loading data in the `DirectInputStream` module: one is the synchronous `loadSync` method, and the other is the asynchronous `loadData` method. While `rawBytesRead` is already being recorded in `loadSync` [here](https://github.com/facebookincubator/velox/blob/main/velox/dwio/common/DirectInputStream.cpp#L148), it is missing the update of this metric in `loadData`. Therefore, we should add the update of `rawBytesRead` in the `loadData` function.